cli/main.c

```diff
@@ -131,7 +131,12 @@ PkStringPtr loadScript(PKVM* vm, const char* path) {
 
 int main(int argc, char** argv) {
 
-  const char* usage = "usage: pocket [-c cmd | file]\n";
+  const char* usage =
+    "usage: pocket [-c cmd | file]\n\n"
+    "The following options are available:\n"
+    "  -c cmd | file    executes a command or a file [TODO]\n"
```

Comment:
Why `TODO` here? we support running a file and an inline command in the `cli` already.

Reply:
Oh okay! Didn't yet test that. Let me remove :) Thanks

Reply:
Also, the `-c cmd` and `file` are 2 different options. They should be in their own lines. Here is the python's help

```
-c cmd : program passed in as string (terminates option list)
file   : program read from script file
```

Reply:
Okay, thanks for informing me.